src/panels/developer-tools/service/developer-tools-service.ts

```diff
@@ -295,7 +299,8 @@ class HaPanelDevService extends LitElement {
     }
   );
 
-  private async _callService() {
+  private async _callService(ev) {
+    const button = ev.currentTarget;
```

Comment:
```suggestion
    const button = ev.currentTarget as HaProgressButton;
```

Reply:
`HaProgressButton` is not exported so we can't use it here. Should I export it as part of the change?